source/class/qx/bom/Label.js

```diff
@@ -286,14 +289,23 @@ qx.Bootstrap.define("qx.bom.Label", {
      * @return {Map} A map with preferred <code>width</code> and <code>height</code>.
      */
     getHtmlSize(content, styles, width) {
+      let cacheKey = this.__getCacheKey(styles, width);
+      let cache = qx.bom.Label.__sizeCache[cacheKey];
+      let size = cache?.sizes[content];
```

Comment:
Sorry, I missed this one the first time. Lines 293-4 should also probably be replaced with use of the getter:

```
let size = this.__getCachedSize(cacheKey, text);
```